### PR TITLE
Fixes Compare items color bug

### LINKED ISSUE #45712: "Compare items" option uses wrong color
# Describe the bug
"Compare items" option uses red color for item with higher coverage and green color for item with lower coverage in section where it lists coverage and encumbrance per specific bodypard. It uses right colours in section where it lists coverage overall. 
# Steps To Reproduce
Compre standard leather gloves and leather fingerless gloves

# Expected behavior
Red for lower coverage and green for higher coverage everywhere.

# Screenshots
Coverage bug
<!-- **If applicable**, add screenshots to help explain your problem.

# Versions and configuration

0.E-7820-g7c1533d (tiles)

### PR DESCRIPTION
**Pull request by YoungBlood01**
CompareColors

#### Summary

SUMMARY: Bugfixes "Compare items now uses correct font colours."

#### Purpose of change

Fixes #45712 - Font colour for 'Coverage' in Encumbrance section of Compare Items will now be Green for higher coverage, red for lower, to match with average coverage above it.

#### Describe the solution

Removed :lower_is_better from Encumbrance-Coverage section to make the font colour match average coverage above it. 

#### Describe alternatives you've considered

Leaving as it is.

#### Testing

Compiles and runs. Spawned multiple armour items and compared - Font colour is now correct. Screenshot attached showing change.

#### Additional context

First PR here so any issues please let me know!

### CODE REVIEW

**Comment by Pupsi-Mupsi** [comment]
Not an issue, but can you add a screenshot.

**Comment by YoungBlood01** [comment]
> Not an issue, but can you add a screenshot.

There you go!

**Review by Aivean — CHANGES_REQUESTED** [review]
Take a look at the failed test:
image
Notice that `Torso` is expected to be on the new line. 
You need to use `iteminfo::no_flags` for Coverage to preserve the old newline behavior, while removing the incorrect `iteminfo::lower_is_better`.

**Comment by YoungBlood01** [comment]
> Take a look at the failed test:
> image
> Notice that `Torso` is expected to be on the new line.
> You need to use `iteminfo::no_flags` for Coverage to preserve the old newline behavior, while removing the incorrect `iteminfo::lower_is_better`.

That's great, thanks for the help.

**Review by Aivean — APPROVED** [review]

File: msvc-full-features/Cataclysm-vcpkg-static.vcxproj [hunk 1]
```diff
@@ -146,4 +146,4 @@
   </ItemGroup>
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets" />
-</Project>
+</Project>
```
Comment (ZhilkinSerg) [inline msvc-full-features/Cataclysm-vcpkg-static.vcxproj:149]:
```suggestion
</Project>

```

**Pull request merged**